api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md

```diff
@@ -35,7 +35,9 @@ This function allows you to customize the toolbar. Depending on the configuratio
 - [*"groupPanel"*](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/groupPanel '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/groupPanel/') 
 - [*"searchPanel"*](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/searchPanel '{basewidgetpath}/Configuration/searchPanel/')
 
-The following code shows how you can customize the toolbar using this function.
+[important]If you use [DevExtreme modules](/concepts/Common/Modularity '/Documentation/Guide/Common/Modularity/'), import the editor's module when you add items other than buttons. 
```

Comment:
> other than buttons

This list must be longer. groupPanel and searchPanel look like TextBox widgets, so TextBox shouldn't require an individual import. There may be other widgets. Besides, it is better to give the note in the end because I don't think many customers use something other than the Button or TextBox in the toolbar.